File: JS-Basics/06-Conditional-Statements-Advanced-Exercise/p05_Journey.js
function journey(input) {
    let buget = Number(input[0]);
    let season = input[1];

    let destination = '';
    let kindVacation = '';

    switch (season) {
        case 'summer':
            if (buget <= 100) {
                destination = 'Bulgaria';
                kindVacation = 'Camp';
                buget *= 0.3;

            } else if (buget <= 1000) {
                destination = 'Balkans';
                kindVacation = 'Camp';
                buget *= 0.4;
            } else if (buget > 1000) {
                destination = 'Europe';
                kindVacation = 'Hotel';
                buget *= 0.9;
            }
            break;
        case 'winter':
            kindVacation = 'Hotel';
            if (buget <= 100) {
                destination = 'Bulgaria';
                buget *= 0.7;
            } else if (buget <= 1000) {
                destination = 'Balkans';
                buget *= 0.8;
            } else if (buget > 1000) {
                destination = 'Europe';
                buget *= 0.9;
            }
            break;
    }

    console.log(`Somewhere in ${destination}`);
    console.log(`${kindVacation} - ${(buget).toFixed(2)}`);
}

journey(["50", "summer"]);
journey(["75", "winter"]);
journey(["312", "summer"]);
journey(["678.53", "winter"]);
journey(["1500", "summer"]);

File: Programming-Basic/06-Conditional-Statements-Advanced-Exercise/p05_Journey.java
import java.util.Scanner;

public class p05_Journey {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

//•	Първи ред – Бюджет, реално число в интервала [10.00...5000.00].
//•	Втори ред –  Един от двата възможни сезона: „summer” или “winter”
        double budget = Double.parseDouble(scanner.nextLine());
        String season = scanner.nextLine();

        Double expenses = 0.0;
        String destination = "";
        String place = "";

        if (budget <= 100) {
            destination = "Bulgaria";

            if (season.equals("summer")) {
                expenses = budget * 0.30;
                place = "Camp";
            } else if (season.equals("winter")) {
                expenses = budget * 0.70;
                place = "Hotel";
            }

        } else if (budget <= 1000) {
            destination = "Balkans";

            if (season.equals("summer")) {
                expenses = budget * 0.40;
                place = "Camp";
            } else if (season.equals("winter")) {
                expenses = budget * 0.80;
                place = "Hotel";
            }

        } else {
            destination = "Europe";

            expenses = budget * 0.9;
            place = "Hotel";
        }
        System.out.printf("Somewhere in %s%n", destination);
        System.out.printf("%s - %.2f", place, expenses);
    }
}
